pkg/controllers/agent/yurtcluster_controller.go

```diff
@@ -128,8 +134,9 @@ func (r *YurtClusterReconciler) reconcile(ctx context.Context, yurtCluster *oper
 			return ctrl.Result{}, err
 		}
 	}
-
-	switch controllersutil.GetNodeType(node, yurtCluster) {
+	nodeType := controllersutil.GetNodeType(node, yurtCluster)
+	r.recorder.Eventf(yurtCluster, corev1.EventTypeNormal, "PrepareConvert", "node %s(type: %s) ready to convert.", node.Name, nodeType)
```

Comment:
I'm worried that events like this will be created over and over again, as the controller is always listening for changes of the YurtCluster object.